.gitlab/kitchen_testing/new-e2e_testing/suse.yml

```diff
@@ -165,19 +116,4 @@ new-e2e-agent-platform-install-script-upgrade6-suse-x86_64:
   parallel:
     matrix:
       - START_MAJOR_VERSION: [6]
-        END_MAJOR_VERSION: [6]
-
-new-e2e-agent-platform-install-script-upgrade7-suse-iot-agent-x86_64:
-  stage: kitchen_testing
-  extends:
-    - .new_e2e_template
-    - .new-e2e_script_upgrade7
-    - .new-e2e_os_suse
-    - .new-e2e_suse_a7_x86_64
-    - .new-e2e_agent_a7
-  variables:
-    FLAVOR: datadog-iot-agent
-  parallel:
-    matrix:
-      - START_MAJOR_VERSION: [7]
         END_MAJOR_VERSION: [7]
```

Comment:
Same question here if this has to be changed

Reply:
Same as previously